core/drivers/stm32_bsec.c

```diff
@@ -624,6 +638,7 @@ static size_t nvmem_layout_count;
 
 TEE_Result stm32_bsec_find_otp_in_nvmem_layout(const char *name,
 					       uint32_t *otp_id,
+					       uint8_t *otp_bit_offset,
```

Comment:
using a `uint8_t` pointer is not really needed, using native `size_t` is ok.
Not a strong opinion.

Reply:
bit offset as limited range => 0...7, 
so uint8_t is enought
I plan to change type of bit_offset in nvmem_layout (I forget to do it)